src/main/java/by/exadel/internship/controller/UserController.java

```diff
@@ -56,4 +58,11 @@ public void restoreUser(@PathVariable UUID userId) {
     public List<UserDTO> getDeletedUserList() {
         return userService.getAllDeleted();
     }
+
+    @SuperAdminAccessControl
+    @PostMapping("/time-slot")
```

Comment:
{userId}/time-slot

Reply:
Use this param like userID even when we have a userDTO with this userID?

Reply:
`userDTO ` has `userId` since we use the same dto for request and response. You implement RESTful API and according it client work with existing resource via resource's id.

Reply:
Done